test/CMakeLists.txt

```diff
@@ -1,44 +1,43 @@
 cmake_minimum_required(VERSION 3.2.2)
 
-# compile google test, google mock
-
+# compile googletest
 include(ExternalProject)
 
 set(SYCLBLAS_TEST ${CMAKE_CURRENT_SOURCE_DIR})
 set(SYCLBLAS_TEST_INCLUDE "${SYCLBLAS_TEST}/include")
 
+# Download and build googletest
+# The following operations are derived from the official advice on building googletest as a subproject,
+# form the following documentation: https://github.com/google/googletest/blob/master/googletest/README.md
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/../cmake/templates/CMakeLists.txt.in ${CMAKE_CURRENT_BINARY_DIR}/googletest-download/CMakeLists.txt)
```

Comment:
Can you split this call a little bit so that the code can be closer to 80 columns?